react-client/src/rootReducers.js

```diff
@@ -2,11 +2,13 @@ import { combineReducers } from 'redux';
 
 import projectReducer from './ProjectIdeas/projectReducer';
 import testReducer from './TestFeature/testReducer';
+import newsReducer from './NewsPage/newsReducer';
 
 const appReducer = combineReducers({
   // all reducers in project
   testReducer,
   projects: projectReducer,
+  newsReducer,
```

Comment:
Jon was using the <name>:<reducer-name> syntax, think it might be a little bit clearer so we don't have to use newsReducer in our mapToState